swarm_test.go

```diff
@@ -19,6 +19,12 @@ package swarm
 import (
 	"context"
 	"encoding/hex"
+	"github.com/ethereum/go-ethereum/common"
```

Comment:
the order of these imports will trigger a linter error, make sure to order this delcaration like so:
```
import (
	"context"
	"encoding/hex"
	"io/ioutil"
	"math/rand"
	"os"
	"path"
	"runtime"
	"strings"
	"testing"
	"time"

	"github.com/ethereum/go-ethereum/common"
	"github.com/ethereum/go-ethereum/crypto"
	"github.com/ethereum/go-ethereum/rpc"
	"github.com/ethersphere/swarm/api"
	"github.com/ethersphere/swarm/sctx"
	"github.com/ethersphere/swarm/swap"
)

```